File: C Programs/ArrayLeftRotation.c
#include<stdio.h>
void left(int arr[],int n);
int main()
{
    int n;
    scanf("%d",&n);
    int arr[n],k;
    for(int i=0;i<n;i++)
    {
        scanf("%d",&arr[i]);
    }
    scanf("%d",&k);
    for(int i=0;i<k;i++)
    {
        left(arr,n);
    }
    for(int i=0;i<n;i++)
    {
        printf("%d ",arr[i]);
    }
}
void left(int arr[],int n)
{
    int temp=arr[0];
    for(int i=0;i<n;i++)
    {
        arr[i]=arr[i+1];
    }
    arr[n-1]=temp;
}

File: Java Programs/ArrayLeftRotation.java
import java.util.*;
public class ArrayLeftRotation 
{
    public static void main(String[] args) 
    {
        Scanner obj=new Scanner(System.in);
        int n=obj.nextInt();
        int arr[]=new int[n];
        for(int i=0;i<n;i++)
        {
            arr[i]=obj.nextInt();
        }
        int k=obj.nextInt();
        for(int i=0;i<k;i++)
        {
            left(arr,n);
        }
        for(int i=n-1;i>=0;i--)
        {
            System.out.print(arr[i]+" ");
        }
    }
    public static void left(int arr[],int n)
    {
        int temp=arr[0];
        for(int i=1;i<n;i++)
        {
            arr[i-1]=arr[i];
        }
        arr[n-1]=temp;
    }  
}